tests/cli/test_cli.py

```diff
@@ -112,15 +120,39 @@ def test_pulling_image_message(self, runner, container_client: ContainerClient):
         # presence of another message which is present when pulling the image.
         assert "download complete" in result.output
 
+    def test_start_already_existing(self, runner, container_client, create_container):
+        # normal start
+        result = runner.invoke(cli, ["start"])
+        assert container_exists(container_client, config.MAIN_CONTAINER_NAME)
+        assert result.exit_code == 1
+        assert "Error" in result.output
+        assert "already exists" in result.output
+
+        # detached start
+        result = runner.invoke(cli, ["start", "-d"])
+        assert container_exists(container_client, config.MAIN_CONTAINER_NAME)
+        assert result.exit_code == 1
+        assert "Error" in result.output
+        assert "already exists" in result.output
+
     def test_start_already_running(self, runner, container_client):
         runner.invoke(cli, ["start", "-d"])
         runner.invoke(cli, ["wait", "-t", "180"])
+
+        # normal start
         result = runner.invoke(cli, ["start"])
         assert container_exists(container_client, config.MAIN_CONTAINER_NAME)
         assert result.exit_code == 1
         assert "Error" in result.output
         assert "is already running" in result.output
 
+        # detached start
+        result = runner.invoke(cli, ["start", "-d"])
+        assert container_exists(container_client, config.MAIN_CONTAINER_NAME)
+        assert result.exit_code == 0
```

Comment:
docs(nit): A clarifying comment on why the behavior differs for `-d` would be helpful here (and/or in the implementation) to understand the rationale.

Reply:
Good point! I added comments with f82e428.
I do agree with the semantics, but also that there isn't much to do from here.
In another iteration on the CLI it would be super interesting to really think through the terminology and semantics (maybe `localstack up` would be better than `start`?).